### DISCUSSION TITLE
Can't reach this page

### DISCUSSION THREAD

**Discussion by labeshops** [lllyasviel/Fooocus#2752] [General]
so I installed focus on my pc and when it first finished, it opened the browser page. I bookmarked the page so I could go back to it after lunch. But now I get "can't reach this page" 127.0.0.1 refused to connect.  err_connection refused

How do I solve this?

**Comment by labeshops** [comment]
I can get it to open the tab if I rerun the .bat file, but when I try to do anything, I get "error - connection errored out". No idea what I am doing wrong :(

**Reply by ILikeToasters** [reply]
Are you closing the CMD window after running the .bat file? Because you do need to leave that going while using it. It normally will launch a new browser window when you run it. But that address should work when it is running.

**Reply by mashb1t** [reply]
Please make sure not to close the cmd window as long as you're using Fooocus and report console log issues as Github issues (if not yet solved).
